javascript/src/main/App.js

```diff
@@ -43,6 +43,7 @@ function App() {
       <Container className="flex-grow-1 mt-5">
         <Switch>
           <Route path="/" exact component={Home} />
+          <Route path="/courses/show/:courseId" component={CourseShow} />
```

Comment:
Line 46 and line 76 appear to be duplicates.

Could we remove one of the two?

Reply:
Line 46  removed